src/App.tsx

```diff
@@ -20,9 +20,9 @@ function App() {
   return (
     <>
       <LoginContext.Provider value={{user, setUser}}>
-        <LoginPage />
+        {/*<LoginPage />*/}
+        <Error404 />
```

Comment:
I recommend putting the `<Menu/>` component here if you can, lmk if it's not working and I can try to help!

Reply:
Yeah it looks like when I have it here instead it won't show up for some reason